FFWEB-1397 Create filter cloud demo

wdutkiewicz-obj opened the pull request:
The demo has been created, but webcomponents reference must be updated after version 3.8 release.

Commit:
FFWEB-1397 Added demo for `ff-filter-cloud`

Commit:
FFWEB-1397 Added style for unresolved components

omikron-fp reviewed (changes requested):
For a demo of the filter-cloud the filter-cloud it is quite small. I'm thinking of a more prominent presentation.
The example in the original issue should be a good inspiration.
https://github.com/FACT-Finder-Web-Components/ff-web-components/issues/21

ghost commented:
> For a demo of the filter-cloud the filter-cloud it is quite small. I'm thinking of a more prominent presentation.
> The example in the original issue should be a good inspiration.
> [FACT-Finder-Web-Components/ff-web-components#21](https://github.com/FACT-Finder-Web-Components/ff-web-components/issues/21)

OK, I guess you mean only a size of the control, not the features it presents? Please, correct me if I'm wrong.

omikron-fp commented:
I mean everything. Component area, font-size, padding, "x". And perhaps even a caption "Selected Filters".
So pretty much like in the screenshot.

Commit:
FFWEB-1397 Visual adjustments to the Filter Cloud demo

Position of Filter Cloud rearranged, improved visibility and unified font, FF Webcomponents reference updated to version 3.8 (first release that contains `ff-filter-cloud`)

ghost commented:
> I mean everything. Component area, font-size, padding, "x". And perhaps even a caption "Selected Filters".
> So pretty much like in the screenshot.

Should be fine right now. Please verify.

omikron-fp commented on ff-filter-cloud/styles.css:
```diff
@@ -0,0 +1,222 @@
+    color: #5C5C5C;
+    background-color: #C7E4EC;
+    width: fit-content;
+    border-radius: 10px;
```
[optional] For consistency `5px` might fit better in the whole picture.

wdutkiewicz-obj replied:
I will leave as it is now. With 5px radius they look like buttons.
